src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```
![image](https://user-images.githubusercontent.com/96409563/216560258-d656305e-d13e-4c3a-b427-d0b0f5235a14.png)